src/components/application_manager/rpc_plugins/rc_rpc_plugin/test/interior_data_cache_test.cc

```diff
@@ -34,67 +34,66 @@
 
 namespace rc_rpc_plugin_test {
 
-class InteriorDataCacheTest : public ::testing::Test  {
-};
+class InteriorDataCacheTest : public ::testing::Test {};
 
-TEST_F(InteriorDataCacheTest, InteriorDataCacheDoesNotContainRandomDataInitialy) {
-    const rc_rpc_plugin::InteriorDataCacheImpl cache;
-    std::string module_type_key = "random_module_type";
-    EXPECT_FALSE(cache.Check(module_type_key));
-    auto retrived_data = cache.Retrive(module_type_key);
-    EXPECT_EQ(retrived_data.getType(), smart_objects::SmartType_Null);
+TEST_F(InteriorDataCacheTest,
+       InteriorDataCacheDoesNotContainRandomDataInitialy) {
+  const rc_rpc_plugin::InteriorDataCacheImpl cache;
+  std::string module_type_key = "random_module_type";
+  EXPECT_FALSE(cache.Check(module_type_key));
+  auto retrived_data = cache.Retrive(module_type_key);
+  EXPECT_EQ(retrived_data.getType(), smart_objects::SmartType_Null);
 }
 
-TEST_F(InteriorDataCacheTest, CheckExstingSubscribed ) {
-    rc_rpc_plugin::InteriorDataCacheImpl cache;
-    const std::string module_type_key = "random_module_type";
-    smart_objects::SmartObject data;
-    data["key"] = "value";
+TEST_F(InteriorDataCacheTest, CheckExstingSubscribed) {
```

Comment:
@LuxoftAKutsan 
- Error in test case name
- Unclear test case name

Reply:
@AByzhynar fixed 197cf55a41c174f5c9809779b2e73d0e404f3a75